### ISSUE TITLE
Mylar keeps adding large amounts of random series by itself

### ISSUE THREAD

**Issue by crusif** [evilhero/mylar#2369]
Hi there,

I’m running into a problem with Mylar, making it practically unusable, hope you can help out!

**Description of the bug**
Up until recently Mylar was running like I wanted it to. About two weeks ago, Mylar started adding series randomly and on it’s own.

What I’ve tried myself:
*Setting Mylar up with no comicvine api-key - Mylar still adds the comic based on Comic ID with no further information
*Check for comics on my autowatch-list - no comics there
*Remove al existing comics from the database and clearing out the wanted-section - also no luck, still comics being added
*Clean install of Mylar - no luck, even with no settings applied (no comic folder, no Post-processing enabled, clear cache-folder) Mylar starts adding random series

I can’t figure out what prompts Mylar to add these series...

**Screenshots/Logs**

Screenshot of my homepage, filled with series I have not added myself, and have no desire in collecting. There seems to be a pattern in the naming of the series, ie “free comic book day”

<img width="1024" alt="Mylar - Home" src="https://user-images.githubusercontent.com/58007320/69321096-c0307f00-0c42-11ea-8892-59f5e35364f3.png">

**Logs - excerpt from the point it started adding series again.**

21-Nov-2019 03:36:27 - INFO    :: mylar.addComictoDB.126 : Thread-40 : Now adding/updating: The Legend of Zelda: Twilight Princess Free Comic Book Day 2017 Edition
21-Nov-2019 03:36:27 - INFO    :: mylar.addComictoDB.179 : Thread-40 : Sucessfully retrieved details for The Legend of Zelda: Twilight Princess Free Comic Book Day 2017 Edition
21-Nov-2019 03:36:27 - INFO    :: mylar.addComictoDB.185 : Thread-40 : Previous version detected as None - seeing if update required
21-Nov-2019 03:36:27 - WARNING :: mylar.validateAndCreateDirectory.1623 : Thread-40 : [DIRECTORY-CHECK] Could not find comic directory: /volume1/homes/admin/Comics/The Legend of Zelda Twilight Princess Free Comic Book Day 2017 Edition (2017)
21-Nov-2019 03:36:27 - INFO    :: mylar.validateAndCreateDirectory.1626 : Thread-40 : [DIRECTORY-CHECK] Creating comic directory (0777) : /volume1/homes/admin/Comics/The Legend of Zelda Twilight Princess Free Comic Book Day 2017 Edition (2017)
21-Nov-2019 03:36:29 - INFO    :: mylar.getImage.3683 : Thread-40 : Attempting to retrieve the comic image for series
21-Nov-2019 03:36:31 - INFO    :: mylar.addComictoDB.299 : Thread-40 : Determined to be a one-shot issue. Forcing Edition to One-Shot
21-Nov-2019 03:36:33 - INFO    :: mylar.addComictoDB.341 : Thread-40 : Sucessfully retrieved issue details for The Legend of Zelda: Twilight Princess Free Comic Book Day 2017 Edition
21-Nov-2019 03:36:39 - INFO    :: mylar.updateissuedata.1098 : Thread-40 : Now adding/updating issues for The Legend of Zelda: Twilight Princess Free Comic Book Day 2017 Edition
21-Nov-2019 03:36:39 - INFO    :: mylar.addComictoDB.401 : Thread-40 : Updating complete for: The Legend of Zelda: Twilight Princess Free Comic Book Day 2017 Edition
21-Nov-2019 03:36:39 - INFO    :: mylar.forceRescan.952 : Thread-40 : [FILE-RESCAN] Now checking files for The Legend of Zelda: Twilight Princess Free Comic Book Day 2017 Edition (2017) in /volume1/homes/admin/Comics/The Legend of Zelda Twilight Princess Free Comic Book Day 2017 Edition (2017)
21-Nov-2019 03:36:39 - INFO    :: mylar.traverse_directories.1408 : Thread-40 : there are 0 files.
21-Nov-2019 03:36:39 - INFO    :: mylar.forceRescan.1504 : Thread-40 : [FILE-RESCAN] Updated the status of 1 issues for The Legend of Zelda: Twilight Princess Free Comic Book Day 2017 Edition (2017) that were not found.
21-Nov-2019 03:36:39 - INFO    :: mylar.forceRescan.1506 : Thread-40 : [FILE-RESCAN] Total files located: 0
21-Nov-2019 03:36:39 - INFO    :: mylar.forceRescan.1615 : Thread-40 : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 0 in an Archived state [ Total Issue Count: 0 / 1 ]
21-Nov-2019 03:36:39 - INFO    :: mylar.addComictoDB.517 : Thread-40 : Sucessfully added The Legend of Zelda: Twilight Princess Free Comic Book Day 2017 Edition (2017) to the watchlist
21-Nov-2019 03:38:10 - INFO    :: mylar.dbUpdate.353 : Thread-26 : Update complete
21-Nov-2019 03:38:11 - INFO    :: mylar.run.2841 : Thread-35 : [FOLDER-CHECK] Checking folder /volume1/homes/admin/Downloads/ for newly snatched downloads
21-Nov-2019 03:38:11 - INFO    :: mylar.traverse_directories.1408 : Thread-35 : there are 0 files.
21-Nov-2019 03:38:11 - WARNING :: mylar.Process.405 : Thread-35 : There were no files located - check the debugging logs if you think this is in error.
21-Nov-2019 03:38:11 - INFO    :: mylar.run.2844 : Thread-35 : [FOLDER-CHECK] Finished checking for newly snatched downloads
21-Nov-2019 03:43:10 - INFO    :: mylar.dbUpdate.353 : Thread-14 : Update complete
21-Nov-2019 03:43:11 - INFO    :: mylar.run.2841 : Thread-36 : [FOLDER-CHECK] Checking folder /volume1/homes/admin/Downloads/ for newly snatched downloads
21-Nov-2019 03:43:11 - INFO    :: mylar.traverse_directories.1408 : Thread-36 : there are 0 files.
21-Nov-2019 03:43:11 - WARNING :: mylar.Process.405 : Thread-36 : There were no files located - check the debugging logs if you think this is in error.
21-Nov-2019 03:43:11 - INFO    :: mylar.run.2844 : Thread-36 : [FOLDER-CHECK] Finished checking for newly snatched downloads
21-Nov-2019 03:48:10 - INFO    :: mylar.dbUpdate.353 : Thread-27 : Update complete
21-Nov-2019 03:48:11 - INFO    :: mylar.run.2841 : Thread-37 : [FOLDER-CHECK] Checking folder /volume1/homes/admin/Downloads/ for newly snatched downloads
21-Nov-2019 03:48:11 - INFO    :: mylar.traverse_directories.1408 : Thread-37 : there are 0 files.
21-Nov-2019 03:48:11 - WARNING :: mylar.Process.405 : Thread-37 : There were no files located - check the debugging logs if you think this is in error.
21-Nov-2019 03:48:11 - INFO    :: mylar.run.2844 : Thread-37 : [FOLDER-CHECK] Finished checking for newly snatched downloads
21-Nov-2019 03:53:10 - INFO    :: mylar.dbUpdate.353 : Thread-21 : Update complete
21-Nov-2019 03:53:11 - INFO    :: mylar.run.2841 : Thread-38 : [FOLDER-CHECK] Checking folder /volume1/homes/admin/Downloads/ for newly snatched downloads
21-Nov-2019 03:53:11 - INFO    :: mylar.traverse_directories.1408 : Thread-38 : there are 0 files.
21-Nov-2019 03:53:11 - WARNING :: mylar.Process.405 : Thread-38 : There were no files located - check the debugging logs if you think this is in error.
21-Nov-2019 03:53:11 - INFO    :: mylar.run.2844 : Thread-38 : [FOLDER-CHECK] Finished checking for newly snatched downloads
21-Nov-2019 03:58:04 - INFO    :: mylar.addComictoDB.126 : Thread-41 : Now adding/updating: Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:04 - INFO    :: mylar.addComictoDB.179 : Thread-41 : Sucessfully retrieved details for Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:04 - INFO    :: mylar.addComictoDB.185 : Thread-41 : Previous version detected as None - seeing if update required
21-Nov-2019 03:58:04 - WARNING :: mylar.validateAndCreateDirectory.1623 : Thread-41 : [DIRECTORY-CHECK] Could not find comic directory: /volume1/homes/admin/Comics/Free Comic Book Day 2015 (Secret Wars) (2015)
21-Nov-2019 03:58:04 - INFO    :: mylar.validateAndCreateDirectory.1626 : Thread-41 : [DIRECTORY-CHECK] Creating comic directory (0777) : /volume1/homes/admin/Comics/Free Comic Book Day 2015 (Secret Wars) (2015)
21-Nov-2019 03:58:06 - INFO    :: mylar.getImage.3683 : Thread-41 : Attempting to retrieve the comic image for series
21-Nov-2019 03:58:09 - INFO    :: mylar.addComictoDB.341 : Thread-41 : Sucessfully retrieved issue details for Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:10 - ERROR   :: mylar.dbUpdate.106 : Thread-28 : Free Comic Book Day 2015 (Secret Wars) failed during a previous add /refresh as it has no Last Update timestamp. Forcing refresh now.
21-Nov-2019 03:58:10 - INFO    :: mylar.dbUpdate.117 : Thread-28 : [8/8] Refreshing :Free Comic Book Day 2015 (Secret Wars) (2015) [81729]
21-Nov-2019 03:58:10 - INFO    :: mylar.addComictoDB.84 : Thread-28 : aliases currently: None
21-Nov-2019 03:58:10 - INFO    :: mylar.validateAndCreateDirectory.1620 : Thread-28 : [DIRECTORY-CHECK] Found comic directory: /volume1/homes/admin/Comics/Free Comic Book Day 2015 (Secret Wars) (2015)
21-Nov-2019 03:58:11 - INFO    :: mylar.run.2841 : Thread-39 : [FOLDER-CHECK] Checking folder /volume1/homes/admin/Downloads/ for newly snatched downloads
21-Nov-2019 03:58:11 - INFO    :: mylar.traverse_directories.1408 : Thread-39 : there are 0 files.
21-Nov-2019 03:58:12 - WARNING :: mylar.Process.405 : Thread-39 : There were no files located - check the debugging logs if you think this is in error.
21-Nov-2019 03:58:12 - INFO    :: mylar.run.2844 : Thread-39 : [FOLDER-CHECK] Finished checking for newly snatched downloads
21-Nov-2019 03:58:13 - INFO    :: mylar.addComictoDB.126 : Thread-28 : Now adding/updating: Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:13 - INFO    :: mylar.addComictoDB.179 : Thread-28 : Sucessfully retrieved details for Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:13 - INFO    :: mylar.addComictoDB.185 : Thread-28 : Previous version detected as None - seeing if update required
21-Nov-2019 03:58:13 - INFO    :: mylar.addComictoDB.223 : Thread-28 : Directory (/volume1/homes/admin/Comics/Free Comic Book Day 2015 (Secret Wars) (2015)) already exists! Continuing...
21-Nov-2019 03:58:15 - INFO    :: mylar.updateissuedata.1098 : Thread-41 : Now adding/updating issues for Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:15 - INFO    :: mylar.addComictoDB.401 : Thread-41 : Updating complete for: Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:15 - INFO    :: mylar.forceRescan.952 : Thread-41 : [FILE-RESCAN] Now checking files for Free Comic Book Day 2015 (Secret Wars) (2015) in /volume1/homes/admin/Comics/Free Comic Book Day 2015 (Secret Wars) (2015)
21-Nov-2019 03:58:15 - INFO    :: mylar.traverse_directories.1408 : Thread-41 : there are 0 files.
21-Nov-2019 03:58:15 - INFO    :: mylar.forceRescan.1504 : Thread-41 : [FILE-RESCAN] Updated the status of 1 issues for Free Comic Book Day 2015 (Secret Wars) (2015) that were not found.
21-Nov-2019 03:58:15 - INFO    :: mylar.forceRescan.1506 : Thread-41 : [FILE-RESCAN] Total files located: 0
21-Nov-2019 03:58:15 - INFO    :: mylar.forceRescan.1615 : Thread-41 : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 0 in an Archived state [ Total Issue Count: 0 / 1 ]
21-Nov-2019 03:58:15 - INFO    :: mylar.addComictoDB.517 : Thread-41 : Sucessfully added Free Comic Book Day 2015 (Secret Wars) (2015) to the watchlist
21-Nov-2019 03:58:15 - INFO    :: mylar.addComictoDB.341 : Thread-28 : Sucessfully retrieved issue details for Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:21 - INFO    :: mylar.updateissuedata.1098 : Thread-28 : Now adding/updating issues for Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:21 - INFO    :: mylar.addComictoDB.401 : Thread-28 : Updating complete for: Free Comic Book Day 2015 (Secret Wars)
21-Nov-2019 03:58:21 - INFO    :: mylar.forceRescan.952 : Thread-28 : [FILE-RESCAN] Now checking files for Free Comic Book Day 2015 (Secret Wars) (2015) in /volume1/homes/admin/Comics/Free Comic Book Day 2015 (Secret Wars) (2015)
21-Nov-2019 03:58:21 - INFO    :: mylar.traverse_directories.1408 : Thread-28 : there are 0 files.
21-Nov-2019 03:58:21 - INFO    :: mylar.forceRescan.1504 : Thread-28 : [FILE-RESCAN] Updated the status of 1 issues for Free Comic Book Day 2015 (Secret Wars) (2015) that were not found.
21-Nov-2019 03:58:21 - INFO    :: mylar.forceRescan.1506 : Thread-28 : [FILE-RESCAN] Total files located: 0
21-Nov-2019 03:58:21 - INFO    :: mylar.forceRescan.1615 : Thread-28 : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 0 in an Archived state [ Total Issue Count: 0 / 1 ]
21-Nov-2019 03:58:21 - INFO    :: mylar.addComictoDB.517 : Thread-28 : Sucessfully added Free Comic Book Day 2015 (Secret Wars) (2015) to the watchlist
21-Nov-2019 03:58:21 - INFO    :: mylar.dbUpdate.353 : Thread-28 : Update complete
21-Nov-2019 04:03:12 - INFO    :: mylar.run.2841 : Thread-18 : [FOLDER-CHECK] Checking folder /volume1/homes/admin/Downloads/ for newly snatched downloads
21-Nov-2019 04:03:12 - INFO    :: mylar.traverse_directories.1408 : Thread-18 : there are 0 files.
21-Nov-2019 04:03:12 - WARNING :: mylar.Process.405 : Thread-18 : There were no files located - check the debugging logs if you think this is in error.
21-Nov-2019 04:03:12 - INFO    :: mylar.run.2844 : Thread-18 : [FOLDER-CHECK] Finished checking for newly snatched downloads
21-Nov-2019 04:03:21 - INFO    :: mylar.dbUpdate.353 : Thread-30 : Update complete
21-Nov-2019 04:05:15 - INFO    :: mylar.addComictoDB.126 : Thread-42 : Now adding/updating: Archie's 65th Anniversary Bash, Free Comic Book Day Edition
21-Nov-2019 04:05:15 - INFO    :: mylar.addComictoDB.179 : Thread-42 : Sucessfully retrieved details for Archie's 65th Anniversary Bash, Free Comic Book Day Edition
21-Nov-2019 04:05:15 - INFO    :: mylar.addComictoDB.185 : Thread-42 : Previous version detected as None - seeing if update required
21-Nov-2019 04:05:15 - WARNING :: mylar.validateAndCreateDirectory.1623 : Thread-42 : [DIRECTORY-CHECK] Could not find comic directory: /volume1/homes/admin/Comics/Archies 65th Anniversary Bash Free Comic Book Day Edition (2006)
21-Nov-2019 04:05:15 - INFO    :: mylar.validateAndCreateDirectory.1626 : Thread-42 : [DIRECTORY-CHECK] Creating comic directory (0777) : /volume1/homes/admin/Comics/Archies 65th Anniversary Bash Free Comic Book Day Edition (2006)
21-Nov-2019 04:05:17 - INFO    :: mylar.getImage.3683 : Thread-42 : Attempting to retrieve the comic image for series
21-Nov-2019 04:05:18 - INFO    :: mylar.addComictoDB.299 : Thread-42 : Determined to be a one-shot issue. Forcing Edition to One-Shot
21-Nov-2019 04:05:21 - INFO    :: mylar.addComictoDB.341 : Thread-42 : Sucessfully retrieved issue details for Archie's 65th Anniversary Bash, Free Comic Book Day Edition
21-Nov-2019 04:05:27 - INFO    :: mylar.updateissuedata.1098 : Thread-42 : Now adding/updating issues for Archie's 65th Anniversary Bash, Free Comic Book Day Edition
21-Nov-2019 04:05:27 - INFO    :: mylar.addComictoDB.401 : Thread-42 : Updating complete for: Archie's 65th Anniversary Bash, Free Comic Book Day Edition
21-Nov-2019 04:05:27 - INFO    :: mylar.forceRescan.952 : Thread-42 : [FILE-RESCAN] Now checking files for Archie's 65th Anniversary Bash, Free Comic Book Day Edition (2006) in /volume1/homes/admin/Comics/Archies 65th Anniversary Bash Free Comic Book Day Edition (2006)
21-Nov-2019 04:05:27 - INFO    :: mylar.traverse_directories.1408 : Thread-42 : there are 0 files.
21-Nov-2019 04:05:27 - INFO    :: mylar.forceRescan.1504 : Thread-42 : [FILE-RESCAN] Updated the status of 1 issues for Archie's 65th Anniversary Bash, Free Comic Book Day Edition (2006) that were not found.
21-Nov-2019 04:05:27 - INFO    :: mylar.forceRescan.1506 : Thread-42 : [FILE-RESCAN] Total files located: 0
21-Nov-2019 04:05:27 - INFO    :: mylar.forceRescan.1615 : Thread-42 : [FILE-RESCAN] I have physically found 0 issues, ignored 0 issues, snatched 0 issues, and accounted for 0 in an Archived state [ Total Issue Count: 0 / 1 ]
21-Nov-2019 04:05:27 - INFO    :: mylar.addComictoDB.517 : Thread-42 : Sucessfully added Archie's 65th Anniversary Bash, Free Comic Book Day Edition (2006) to the watchlist

**Complete log since startup last night**
[mylar.log](https://github.com/evilhero/mylar/files/3873807/mylar.log)

**Environment:**
 - OS: DSM 6.2.2-24922 Update 4
 - Branch / commit: Master/5ffe8fbd3080e11884c6d11ad3aacbc9f6a73cb3

**Comment by evilhero** [comment]
It looks like you have more than one instance of Mylar running at the same time for some reason - as the first few lines of your log file are duplicated right after each other. Once the daemon fires off, the logging goes back to the one log lines, but that's because it doesn't monitor the other instances.

Have you recently tried to start Mylar from a command prompt and didn't kill it properly ?

**Comment by crusif** [comment]
I’ve only started mylar via package center, but I did encounter some “failed to run package service”-errors. After some tries, it started working. The log is from the first startup after a fresh install - after uninstalling the package, and reinstalling through the package center and updating to the latest commit in the master-branche.

**Comment by evilhero** [comment]
It still looks like something was running even after the uninstall. 

Mylar will only add new series to your watchlist via manual methods. 

There are 3 automated ways of adding series:
- the mass import function (it would have to be running in the background)
- the maintenance mode being enabled and being pulled from a json file in the root of the mylar directory.
- the auto-watch option from the pull-list.

Were you doing any of the above prior to reinstalling everything? Are you using the same config.ini file or a new, blank one ? 

Have you tried breaking out into a command line and just doing a simple ```ps -xg | grep python``` and seeing if mylar is listed therein ?

**Comment by crusif** [comment]
Before this whole thing started, I added some comics to auto-watch, as Comicvine wasn’t up to date.

When I reinstalled Mylar I used a brand new config-file.

Need some direction on the command line. Not afraid to try it, just don’t have the experience/knowledge. I successfully ssh’ed into the Synology and pasted your suggestion, but this didn’t prompt a list of running processes.

Some guidance is very welcome!
